### DISCUSSION TITLE
feedback newsletter Occultish

### DISCUSSION THREAD

**Discussion by Juliettervrd** [SecretDecks/Contentwriting#25] [Feedback]
Very great work on the newsletter guys! 
I have some short feedbacks : 

1. I think we would need to add a concrete call to action at the end of the newsletter - what do we want the artists to do after they red this, if they want to take part in the project? What are the next steps to onboard them and how can we get them moving forward with us? 

2. I would recommend to insist more on the beginning of the NL what are the advantages the artists would get while collaborating to the project. I think the technical part should be as simple and short as possible, and artists should be able to understand from the beginning what is the value added of the project, and how the project could bring value to them. 

What do you guys think? [Occultish newsletter](https://github.com/SecretDecks/Contentwriting/blob/main/SecretDecks%C2%AE%20Newsletter%20to%20Occultish.md)

**Comment by DanM3rcurius** [comment]
love it! thanks for the feedback @Juliettervrd 🙏🏿 
I'll edit it first thin in the morning!
Thought? @evolayak

**Comment by evolayak** [comment]
Yep! Brava, @Juliettervrd - thank you :)

**Comment by DanM3rcurius** [comment]
will start editing this now
